### DISCUSSION TITLE
sleep()

### DISCUSSION THREAD

**Discussion by WojciechGw** [picocomputer/community#72] [Q&A] [answered]
Is the sleep() function available?

**Comment by netzerohero** [comment]
I quickly tried sleep(5) in a hello-world in my cc65 environment; I included <unistd.h> header file.  

https://cc65.github.io/doc/funcref.html#ss3.246

Compiles ok; but linker ld65 doesn't find sleep().  So must not be in the standard library.  

I didn't look any further than that. Sorry I can't be of more help ....

**Comment by rumbledethumps** [comment, marked as answer]
try this:
https://github.com/picocomputer/cc65/commit/2138f3c902f2bcbe538a62fae5e7e36c36ea2c2a
https://github.com/picocomputer/rp6502/commit/8f02539d8a6255fd443b77017fc2ad38a6b0bb87

**Reply by WojciechGw** [reply]
thank You both :)
@netzerohero I was stuck in the same place
@Rumbledethumps Your hint gave me the desired effect - sleep() works now
